packages/dds/ink/src/snapshot.ts

```diff
@@ -55,7 +55,9 @@ export class InkData {
 	 * {@inheritDoc IInk.getStroke}
 	 */
 	public getStroke(key: string): IInkStroke {
-		return this.strokes[this.strokeIndex[key]];
+		// TODO Non null asserting, why is this not null?
```

Comment:
You have a TODO here.
I don't know that we can be sure that these are not undefined. But it can be okay that a bad key results in throw. So, we can say that.